android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerPackage.java

```diff
@@ -35,7 +36,7 @@ public ReactShadowNode createShadowNodeInstance() {
 
     @Override
     public Class getShadowNodeClass() {
-      return ReactShadowNode.class;
+      return ReactShadowNodeImpl.class;
```

Comment:
Instead of doing this I'd prefer to change the baseclass of ViewManager and use ReactViewManager instead. It doesn't make much of a difference because views won't be instantiated anyways but may prevent similar incompatibility issues in the future